spec/requests/trucks_request_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "Trucks", type: :request do
+
```

Comment:
copy the get index spec and put it here too i think
https://github.com/mmmmaximus/rails-truck-logistics/pull/10/files#diff-c2cac94b30c0cdf06fa28350cd4f514a887b5a12aba4fdd55621dbe3d66eb44dR5

We will go over what these tests are doing later on

Reply:
what does that mean?

Reply:
copy the spec, its a test to ensure our controllers remain working and dont unintentionally break